### PR TITLE
doc/mgr/ansible.rst: fix typo

### PR DESCRIPTION
**Pull request by Rjerk**
Signed-off-by: Lan Liu <liulan@umcloud.com>

- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

### CODE REVIEW

File: doc/mgr/ansible.rst [hunk 1]
```diff
@@ -42,12 +42,12 @@ Configuration
...
-A client certificate and a key files should be provided by the Administrator of the Ansible Runner Service for each manager node.
-This files should be copied in each of the manager nodes with read access for the ceph user.
+A client certificate and a key file should be provided by the Administrator of the Ansible Runner Service for each manager node.
+These files should be copied in each of the manager nodes with read access for the ceph user.
 The destination folder for this files and the name of the files must be the same always in all the manager nodes,
-althought the certificate/key content of this files logically will be different in each node.
+although the certificate/key content of these files logically will be different in each node.
 
-Configuration must be set once the module is enabled by first time.
+Configuration must be set once the module is enabled by the first time.
```
Comment (tchaikov) [inline doc/mgr/ansible.rst]: probably s/once/when/ and s/by/for/?

Comment (Rjerk) [reply doc/mgr/ansible.rst]: done

**Comment by sebastian-philipp** [comment]
jenkins retest this please

**Comment by jmolmo** [comment]
Thank you @Rjerk !!

**Pull request merged**